census/templates/census/pending_list.html

```diff
@@ -14,11 +14,11 @@ <h1>Pending Events</h1>
       </tr>
       {% for event in event_list %}
         <tr style="border: 3px solid #dddddd; text-align: left">
-          <td><b><a href="/{{ event.id }}/update">{{ event.title }}:</a></b></td>
+          <td><b><a href="/event/{{ event.id }}/update">{{ event.title }}:</a></b></td>
```

Comment:
@n8snyder isn't there a template filter to help create URLs? Like to add the base/site url?